appveyor.yml

```diff
@@ -0,0 +1,6 @@
+version: 1.0.{build}
```

Comment:
Sets the build version number. This was exported from appveyor, and could probably change.